.github/workflows/maven.yml

```diff
@@ -0,0 +1,38 @@
+name: antlr4 project CI
+
+on:
+  push:
+    branches: 
+     - master
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+          ref: ${{ github.event.pull_request.head.sha }}
+    - name: Set up JDK 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: clean verify
+      run: mvn -e --no-transfer-progress clean verify
```

Comment:
I will be disabling this soon, which is why I have created the other runs below. `clean verify` will be failing once I remove the ANTLR2 deps.

Reply:
You can disable all optional CI checks to speed up the process.  All issues can be resolved later, when merging to the main branch.

Reply:
Since I will be writing so much similar code in the visit methods, I would hate to repeat something that will end up being a violation and have to fix 200 or more of the same thing :)